themes/ddbasic/sass/components/view/view.scss

```diff
@@ -0,0 +1,574 @@
+@import '../../base.scss';
```

Comment:
The usage of !important seems to be a popular approach within this document. For a base theme these usages should be avoided if at all possible.